### DISCUSSION TITLE
ESP8266 ------ LC Technology 5V 4 Channel Relay Board (LC-ESP01-4R-5V)

### DISCUSSION THREAD

**Discussion by Abbruscato** [arendst/Tasmota#20367] [Support request]
Good morning, I just installed the firmware on an esp8266 that manages 4 5v relays.
Everything configured but I don't know how to do the following:

After a power failure, the programmed functions should resume upon restoration.
How do you set up this feature?
Thank you

**Comment by sfromis** [comment]
That was a bad board to buy, has caused lots of pain, as it does not allow Tasmota full control.

You may be able to do something with `Power2#boot` rules to pick up initial power state as seen by Tasmota, and combine that with the existing rules suggested on the [template page](https://templates.blakadder.com/LC-ESP01-4R-12V.html) to replicate Tasmota power state to the commands for the other MCU controlling the board.

**Comment by Abbruscato** [comment]
Hi, I tried playing with the rules and Power2#boot, but to no avail.
I wanted to thank you for the advice.
Which 4 relay board do you recommend that can be fully managed by Tasmota?
Thank you

**Reply by barbudor** [reply]
I would suggest this one : https://templates.blakadder.com/robotdyn_ESP32R4.html
